charmcraft.yaml

```diff
@@ -56,8 +56,11 @@ requires:
   logging:
     interface: loki_push_api
   ingress:
-    interface: ingress
+    interface: traefik_route
     limit: 1
+    description: |
+      Adds an ingressed endpoint used for cross-model communication.
```

Comment:
```suggestion
      Ingress integration for Tempo server and Tempo receiver endpoints, so that cross-model workloads can send their traces to Tempo through the ingress.
```

Reply:
Done, thanks!